common/metrics/tags.go

```diff
@@ -197,6 +199,10 @@ func VisibilityTypeTag(value string) Tag {
 	return &tagImpl{key: visibilityTypeTagName, value: value}
 }
 
+func ResourceExhaustedCauseTag(cause enumspb.ResourceExhaustedCause) Tag {
+	return &tagImpl{key: "resource_exhausted_cause", value: cause.String()}
```

Comment:
All other keys are `const` in this file. I am generally ok to use ad-hoc strings if they are used just once (this case) but for consistency I would prefer `const`.

Reply:
Why you added this here between two visibility related sections? Why not add it at the end of the file?